python/llm/example/CPU/HF-Transformers-AutoModels/Model/qwen-vl/README.md

```diff
@@ -0,0 +1,88 @@
+# Qwen-VL
```

Comment:
For the places where we mention the general model family, let's use Qwen-VL; while when we mention the specific model in this model family, let's use Qwen-VL-Chat

Reply:
So here, we should have

`For illustration purposes, we utilize the [Qwen/Qwen-VL-Chat](https://huggingface.co/Qwen/Qwen-VL-Chat) as a reference Qwen-VL model.`